src/main/resources/profiles/harvard.conf

```diff
@@ -1,6 +1,6 @@
 endpoint = "http://vcoai.lib.harvard.edu/vcoai/vc"
 metadataPrefix = "mods"
-outputDir = "/Users/scott/harvard_i3_output"
+outputDir = "/Users/scott/i3_harvard_harvest"
```

Comment:
Weren't we talking about keeping file paths outside of configuration and in command line arguments instead?

Reply:
Sorry, this was just an oversight.